### DISCUSSION TITLE
How to import all objects from a target pipeline ?

### DISCUSSION THREAD

**Discussion by ThomasFaria** [InseeFrLab/ESA-Nowcasting-2023#56] [Q&A] [answered]
How to import all objects from a target pipeline ?

**Comment by antoine-palazz** [comment, marked as answer]
You can run the ```targets::tar_load_everything()``` function to import all objects defined in the pipeline into the current environment. However, similar to Python, this approach can lead to naming conflicts and should be used with caution.
It is generally better to selectively import only the objects that are needed from the pipeline using ```targets::tar_read()``` function. This approach is more explicit, reduces the risk of naming conflicts, and makes the code easier to read and maintain.
